pulp_rpm/app/downloaders.py

```diff
@@ -0,0 +1,49 @@
+from logging import getLogger
+from aiohttp.client_exceptions import ClientResponseError
+from pulpcore.plugin.download import HttpDownloader
+from urllib.parse import urljoin
```

Comment:
Please check other files or pep https://www.python.org/dev/peps/pep-0008/#imports to sort imports appropriately.
We usually leave a newline between different categories.